### ISSUE TITLE
Integration of ChronOntology

### ISSUE THREAD

**Issue by florianthiery** [alexbrandsen/timeperiod2daterange#1]
Is it possible to integrate https://chronontology.dainst.org ?

I could help, if you want ;-) Because I was a developer in this project ;-)

**Comment by alexbrandsen** [comment]
That would be useful! I think it's possible, I'd just need some way to access the time period list within chronontology. 

Could you write a query on the API that fetches all time periods for a certain area(s)? I guess you need to specify https://gazetteer.dainst.org/place/2085493 as the "spatiallyPartOfRegion" to get the Netherlands for example?

**Comment by florianthiery** [comment]
I will try to figure it out (maybe after my holidays). But I think this minion can profit from more sources :-)

**Comment by alexbrandsen** [comment]
Definitely! Especially when we integrate multiple languages :)

**Comment by florianthiery** [comment]
we should have a chat about that in september :-)

**Comment by florianthiery** [comment]
it could be "another side of the coin" to Allards and my Alligator approach, how to come to "absolute fuzzy" dates

**Comment by alexbrandsen** [comment]
Yeah that's something I ran into, almost all dates in archaeology are fuzzy, but I had to make some hard choices to get to non-fuzzy dates to search on... A chat in September sounds like a great plan!
